_posts/2022-06-13-is-my-ai-discriminatory/refer.bib

```diff
@@ -0,0 +1,59 @@
+@misc{gawthrop_color_2022,
```

Comment:
When I knit the blog post Rmd, it seems like the references aren't showing up properly, I can't seem to figure out why